packages/@webex/plugin-meetings/src/roap/index.ts

```diff
@@ -170,7 +170,7 @@ export default class Roap extends StatelessWebexPlugin {
           locusSelfUrl: meeting.selfUrl,
           mediaId: options.mediaId,
           correlationId: options.correlationId,
-          audioMuted: meeting.isAudioMuted(),
```

Comment:
we should really do the same for video too. It's the same MuteState class that manages video mute and it's the same Locus API that we use, so the same bug exists for video, although I'm not sure if you can have video mute on  entry in webex, but anyway, we should have the same code for audio and video

Reply:
I've made these changes @marcin-bazyl 

Kindly check. However this may not be the case until in the future we get a case for Remote Video Mute & On Join Video Mute. We have neither use cases right now in the Meetings.